### ISSUE TITLE
[fixed: CTR 1.5.9b8] small button mode on Windows + Tile Tabs toolbar button (see #47)

### ISSUE THREAD

**Issue by Imold** [Aris-t2/ClassicThemeRestorer#164]
#47 Tile Tabs now provides a button id for its toolbar button. Changing

`#navigator-toolbox #nav-bar .toolbarbutton-menubutton-button .toolbarbutton-icon,`
to
`#navigator-toolbox #nav-bar :not(#tiletabs-buttonmenu) .toolbarbutton-menubutton-button .toolbarbutton-icon,`
will resolve the conflict.

**Comment by Aris-t2** [comment]
Sorry, that code does not change anything for me.

The problem is this: tile tabs uses a toolbar button within a toolbar button. The first one offers the mentioned id, but the second one does not have any id, but uses the ".toolbarbutton-menubutton-button". Until it offers a proper id, there is no way to exclude it from the above code without breaking other menu buttons.

**Comment by Aris-t2** [comment]
Any new information about this Imold ?

**Comment by Imold** [comment]
Have you tried replacing 
`#navigator-toolbox #nav-bar .toolbarbutton-menubutton-button .toolbarbutton-icon,`
with
`#navigator-toolbox #nav-bar > .toolbarbutton-menubutton-button > .toolbarbutton-icon,
#navigator-toolbox #nav-bar  .toolbarbutton-menubutton-button:not(#tiletabs-buttonmenu) .toolbarbutton-menubutton-button .toolbarbutton-icon,`
?

**Comment by Aris-t2** [comment]
You can't use .toolbarbutton-menubutton-button:not(#tiletabs-buttonmenu) .toolbarbutton-menubutton-button in a row.

The marked toolbar button needs an id to be excluded from CTR code.

![1](https://cloud.githubusercontent.com/assets/6013569/13711583/ae5bcbd2-e7bf-11e5-8c39-775d2ccf5a01.png)

Lets say it would have an id like "#tiletabs-subbutton", then this could would work

```
#navigator-toolbox #nav-bar .toolbarbutton-menubutton-button:not(#tiletabs-subbutton) .toolbarbutton-icon,
```

**Comment by Aris-t2** [comment]
I have contacted TT dev to add a subbutton id like mentioned above. Depends on his reaction and how fast he can implement a subbutton id, this conflict might get resolved soon.

See: https://addons.mozilla.org/addon/tile-tabs/reviews/

**Comment by Aris-t2** [comment]
Fixed on latest beta/1.5.9 release.
